src/viur/core/utils/json.py

```diff
@@ -47,6 +47,9 @@ def preprocess(obj: t.Any) -> t.Any:
         elif isinstance(obj, (list, tuple)):
             return tuple(ViURJsonEncoder.preprocess(value) for value in obj)
 
+        elif hasattr(obj, "boneMap"):  # SkeletonInstance
```

Comment:
I think this test is necessary because of circular imports. Can you give a hint in form of a comment. I wonder if there isn't a better method, like inspect.